opencog/query/PatternMatchEngine.cc

```diff
@@ -164,8 +165,8 @@ bool PatternMatchEngine::variable_compare(const Handle& hp,
 
 	// Make a record of it.
 	LAZY_LOG_FINE << "Found grounding of variable:";
-	logmsg("$$ variable:    ", hp);
-	logmsg("$$ ground term: ", hg);
+	logmsg("$$ variable:", hp);
+	logmsg("$$ ground term:", hg);
```

Comment:
argh... the whitspace was there to explicitly align the two, so make it easier to read!  Can you put the whitespace back?